src/btree/bt_vrfy.c

```diff
@@ -275,8 +275,12 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_CELL_UNPACK *unpack,
 
     /*
      * Open a history store cursor positioned at the end of the data store key (the newest record)
-     * and iterate backwards until we reach a different key or btree.
+     * and iterate backwards until we reach a different key or btree. We need to do this in a
```

Comment:
Can reword the comment as follows:
we need to do this in a snapshot --> we need to do this while holding a snapshot.